docs/design/2021-04-26-lock-view.md

```diff
@@ -35,11 +35,11 @@ Several tables will be provided in `information_schema`. Some tables has both lo
 
 | Field      | Type       | Comment |
 |------------|------------|---------|
-|`TRX_ID`    | `unsigned bigint`   | The transaction ID (aka. start ts) |
-|`TRX_STARTED`|`time`| Human readable start time of the transaction |
-|`DIGEST`|`text`| The digest of the current executing SQL statement |
-|`SQLS` | `text` | A list of all executed SQL statements' digests |
-|`STATE`| `enum('Running', 'Lock waiting', 'Committing', 'RollingBack')`| The state of the transaction |
+| `TRX_ID`    | `unsigned bigint`   | The transaction ID (aka. start ts) |
+| `TRX_STARTED`|`time`| Human readable start time of the transaction |
+| `DIGEST`|`text`| The digest of the current executing SQL statement |
+| `ALL_SQLS` | `text` | A list of all executed SQL statements' digests |
```

Comment:
The original `SQLS` doesn't looks understandable.
Maybe `ALL_SQLS` is still kind of confusing. Maybe it's better to name it something like `ALL_SQL_DIGESTS`, `FINISHED_SQL_DIGESTS` 🤔

Reply:
This might be too big for this sprint, but:
Would it be better if we make them a seperated table? eg. TRX_STATEMENT.
| Field      | Type       | Comment |
|------------|------------|---------|
| TRX_ID | unsigned bigint| |
| digest | text | |
| index | | This sql is the `index`-th statement in this transaction |

And we can probably add more fields here, eg. each statements's execution time, etc.

Reply:
It looks logically correct to split it to another table, but I'm afraid it's too complicated, and it would be hard to get consistent data from this table and TIDB_TRX when the user wants to join them. It would also be kind of difficult to attach the information to the `DEADLOCK` table like how we designed before.
What's your opinion? @youjiali1995 @cfzjywxk 

By the way, I think we need to implement it in this sprint since we've planed so. Actually, I'm about to start trying it.

Reply:
Currenly maybe we could do it in the simple way.